extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+/**
+ * Extra health checks controller.
+ *
+ * Currently supports requests for getting the status of database clusters
+ */
+class HealthController extends WikiaController {
+
+	private $current = '';
+
+	private $clusters = [ ];
```

Comment:
Those empty brackets look strange :japanese_ogre: (It's just a comment - don't take it personally, please :) Feel free to leave it as it is or change it)